sysbuild/Kconfig.netcore

```diff
@@ -135,18 +137,19 @@ config NETCORE_802154_RPMSG_BOARD
 	string
 	default "nrf5340dk_nrf5340_cpunet" if ($(BOARD) = "nrf5340dk_nrf5340_cpuapp" || $(BOARD) = "nrf5340dk_nrf5340_cpuapp_ns")
 	default "thingy53_nrf5340_cpunet" if ($(BOARD) = "thingy53_nrf5340_cpuapp" || $(BOARD) = "thingy53_nrf5340_cpuapp_ns")
+	default "nrf7002dk_nrf5340_cpunet" if ($(BOARD) = "nrf7002dk_nrf5340_cpuapp" || $(BOARD) = "nrf7002dk_nrf5340_cpuapp_ns")
```

Comment:
Is it correct to support nRF7002 with 802154 rpmsg image?

Reply:
@LuDuda? I don't see the reason why not. Having Wi-Fi and Thread without BLE sounds sane to me for some use cases.

Reply:
This is correct configuration i.e. nRF7002 + 802154 rpmsg image, we do have a client that is going to use it. Though, normal case is hci rpmsg only.

Reply:
So.. nRF7002 + 802154 would mean Wi-Fi + Thread - BLE. While i can see this configuration valid, we do not have any use-case at least in Matter for this.